file_system.go

```diff
@@ -332,7 +419,8 @@ func parseLIST(entry string, loc *time.Location, skipSelfParent bool) (os.FileIn
 
 	matches := lsRegex.FindStringSubmatch(entry)
 	if len(matches) == 0 {
-		return nil, ftpError{err: fmt.Errorf(`failed parsing LIST entry: %s`, entry)}
+		info, err := parseDirLIST(entry, loc, skipSelfParent)
```

Comment:
Can you add tests for this additional parsing? Maybe some unit tests similar to TestParseMLST() ?